operations.go

```diff
@@ -205,6 +205,39 @@ func (c CmdS3Copy) Resolve() *CommandDefinition {
 }
 func s3CopyFactory() Command { return CmdS3Copy{} }
 
+type cmdS3SyncBase struct {
+	ExcludeFilter string `json:"exclude"`
+	MaxRetries    int    `json:"max_retries"`
+}
+
+type CmdS3Push struct {
+	cmdS3SyncBase
```

Comment:
I would just duplicate the fields, as it's difficult/impossible to construct this except by serializing it, and I think having shrub be a good API from other go packages is useful.

Reply:
Done.